src/Guider.cpp

```diff
@@ -168,13 +168,11 @@ void logic()
           bed.setled(!bed.getLed());
           bedTimer.start();
         }
-        if (bedTimer.finished())
-        {
-          bed.setled(0);
-        }
-      } else if (!bedTimer.finished()) 
+      }
+      bedTimer.start(); 
```

Comment:
should be in the pressure sensor if

Reply:
you could also do with a bed.lock() before doing anything with bed and a bed.unlock() when you're done doing things with bed.